### PR TITLE
Rebased with development branch and added validation to form

### PR DESCRIPTION
**Pull request by Saif807380**

### CODE REVIEW

File: src/views/pages/CLView/ApplyNowModal.js [hunk 1]
```diff
@@ -136,8 +137,8 @@ export default function ApplyNowModal() {
               fullWidth
               name="phone"
               onChange={handleChange}
-              validators={['required']}
-              errorMessages={['This is a required field']}
+              validators={['required', 'matchRegexp:^[+]*[(]*[+]{0,1}[0-9]{1,3}[)]{0,1}[-s./0-9]*$']}
+              errorMessages={['This is a required field', 'Please enter a valid contact number']}
             />
```
Comment (Abhishek-kumar09) [inline src/views/pages/CLView/ApplyNowModal.js:142]: Impressive work @Saif807380, 
We can nest a small dropdown (`<SelectValidator />` and this `<TextValidator>`) inside a fragment or grid. To give the user an option to select the country code.
For now we could map through an array of 1 to 100, (with default value 91) or use https://ipstack.com/

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Everything else is great, just a minor suggestion.

**Comment by Saif807380** [comment]
Thanks for the review. I have added the country code dropdown. [Link to preview](https://5f6b48ba14c37100073c3389--eloquent-turing-542749.netlify.app/campusLeaders). I have used an array for it as using [ipstack.com](https://ipstack.com) would require getting an account and an API key.

**Review by Abhishek-kumar09 — APPROVED** [review]
Thanks for your work, its looking great!

**Review by KeenWarrior — APPROVED** [review]

**Comment by KeenWarrior** [comment]
Thanks for great work @Saif807380

**Pull request merged**